pmd-core/src/main/java/net/sourceforge/pmd/lang/document/CpdCompat.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.document;
+
+import net.sourceforge.pmd.cpd.SourceCode;
+import net.sourceforge.pmd.lang.BaseLanguageModule;
+import net.sourceforge.pmd.lang.Language;
+import net.sourceforge.pmd.lang.LanguageVersion;
+
+/**
+ * Compatibility APIs, to be removed before PMD 7 is out.
+ */
+@Deprecated
+public final class CpdCompat {
```

Comment:
Since I want to remove it before 7.0.0, I think it's correct not to mark it `@DeprecatedUntil700`. My understanding is that this annotation marks an API that should not be removed before 7.0.0-beta (or however we call the release that has these compatibility apis)

Reply:
ok, I guess it's anyway internal and no use as an API :)